src/js/background/context-menu.js

```diff
@@ -567,6 +571,8 @@ browser.contextMenus.onClicked.addListener(async (info, tab) => {
         category: "Extension: Context Menu",
         action: "click",
         label: "context-menu-relay-manage-aliases",
+        dimension5: await getBrowser(),
+        imension7: "add-on",
```

Comment:
Happened to see this just now, thought I'd call it out just in case:

```suggestion
        dimension7: "add-on",
```